showCrime/showCrime/settings/production.py

```diff
@@ -0,0 +1,8 @@
+from .base import *
+
+# Email config, ala https://docs.webfaction.com/software/django/getting-started.html?highlight=django%2520email#configuring-django-to-send-email-messages
+ADMIN = ((env('ADMIN_USER'),env('ADMIN_EMAIL')))
+EMAIL_HOST = env('EMAIL_HOST')
```

Comment:
Fallback to Django's defaults: https://docs.djangoproject.com/en/2.1/ref/settings/#email-host for all of the email variables.

Reply:
I'm all for using the sane defaults but I'm not familiar with django-environ. For example, how do we structure the email settings so it picks up django's defaults without having to duplicate the django defaults?

```python
# e.g. avoid doing this
ADMIN = ((env('ADMIN_USER', ''),env('ADMIN_EMAIL', '')))
```

Reply:
You have to specify the defaults, which I am fine with. I find explicit settings to be preferable to implicit settings.

It's also worth noting that, if we move to AWS, we must use [SES](https://aws.amazon.com/ses/) to send email since ports 25 and 465 are blocked on EC2 instances to avoid hosting spam servers. Given that, nearly all of the settings in this PR may not be needed.

Reply:
I really want to use Docker. If we aren’t going that route, then yes, what you have is the way to go.

Sorry for being contrarian. Let’s move forward with this for now.

Reply:
Thanks for being flexible. I'm all for docker too, just looking for the easiest path. Now that the Docker stuff is ready, I'm going to close this PR :)